src/targets/__tests__/npm.test.ts

```diff
@@ -0,0 +1,129 @@
+import { parseVersion } from '../../utils/version';
+import { getPublishTag, getLatestVersion } from '../npm';
+
+function setNpmToken() {
+  process.env.NPM_TOKEN = 'test-token';
+}
```

Comment:
this never gets cleaned up -- there should be some context-like assignment to a global